packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -137,20 +146,22 @@ List<String> _pubCommand(List<String> arguments) {
 }
 
 /// The full environment used when running pub.
-Map<String, String> get _pubEnvironment => <String, String>{
+Map<String, String> _pubEnvironment(String pubContext) => <String, String>{
```

Comment:
this should probably be renamed to make it clearer that it now does something rather than is something. e.g. `_createPubEnvironment` or something.

Reply:
ack